### PR TITLE
data: Dimple-Choudhary

### PR DESCRIPTION
**Pull request by Dimple-Choudhary**
* Added my profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3332"></a>

### CODE REVIEW

**Review by ChinmayMhatre — DISMISSED** [review]
Looks good!

File: data/Dimple-Choudhary.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Dimple choudhary",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer",
+  "avatar": "https://github.com/Dimple-Choudhary.png",
+}"links": [
```
Comment (Pradumnasaraf) [inline data/Dimple-Choudhary.json:7]:
```suggestion
 "links": [
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There is an extra **Curly brackets** `}` in the file. Please fix that. If you need any help feel free to reach out.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**